### ISSUE TITLE
Crash with Primitive Mobs

### ISSUE THREAD

**Issue by xbony2** [Glitchfiend/BiomesOPlenty#1301]
We seem to keep getting this crash, especially right after a player dies from an explosion from a Rocket Creeper. I don't know which mod's fault this is. ([see also](https://github.com/Daveyx0/PrimitiveMobs/issues/115))
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.10.0-1.12.2.jar)
  F5FixLoadingPlugin (f5fix-1.1.jar)
  EnderCorePlugin (EnderCore-1.12.2-0.5.37.jar)
  OpenEyePlugin (OpenEye-1.12.2-0.8.jar)
  BedPatch (bedpatch-2.2-1.12.2.jar)
  AstralCore (astralsorcery-1.12.2-1.9.4.jar)
  OpenModsCorePlugin (OpenModsLib-1.12.2-0.11.5.jar)
  CTMCorePlugin (CTM-MC1.12.2-0.3.2.18.jar)
  LoadingPlugin (Quark-r1.4-125.jar)
  Incorporeal Tweaker (incorporeal-1.0.2.jar)
  LoadingPlugin (HardcoreDarkness-MC1.12.2-2.0.jar)
  TransformerLoader (OpenComputers-MC1.12.2-1.7.2.67.jar)
  SmoothWaterCore (SmoothWater-0.1.4.jar)
  BetterFoliageLoader (BetterFoliage-MC1.12-2.1.10.jar)
Contact their authors BEFORE contacting forge

// On the bright side, I bought you a teddy bear!

Time: 8/18/18 4:03 PM
Description: Ticking entity

java.lang.IllegalArgumentException: Cannot get property PropertyEnum{name=variant, clazz=class biomesoplenty.common.block.BlockBOPGrass$BOPGrassType, values=[spectral_moss, overgrown_stone, loamy, sandy, silty, origin, overgrown_netherrack, daisy, mycelial_netherrack]} as it does not exist in BlockStateContainer{block=minecraft:air, properties=[]}
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201)
	at biomesoplenty.common.block.BlockBOPGrass.getSoundType(BlockBOPGrass.java:123)
	at net.minecraft.entity.Entity.func_180429_a(Entity.java:1068)
	at net.minecraft.entity.Entity.func_70091_d(Entity.java:952)
	at net.minecraft.entity.EntityLivingBase.func_191986_a(EntityLivingBase.java:1949)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2384)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
	at net.minecraft.entity.monster.EntityMob.func_70636_d(EntityMob.java:45)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2170)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(EntityMob.java:50)
	at net.minecraft.entity.monster.EntityCreeper.func_70071_h_(EntityCreeper.java:172)
	at net.daveyx0.primitivemobs.entity.monster.EntityPrimitiveCreeper.func_70071_h_(EntityPrimitiveCreeper.java:66)
	at net.daveyx0.primitivemobs.entity.monster.EntityRocketCreeper.func_70071_h_(EntityRocketCreeper.java:128)
	at net.minecraft.world.World.func_72866_a(World.java:1990)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1952)
	at net.minecraft.world.World.func_72939_s(World.java:1756)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201)
	at biomesoplenty.common.block.BlockBOPGrass.getSoundType(BlockBOPGrass.java:123)
	at net.minecraft.entity.Entity.func_180429_a(Entity.java:1068)
	at net.minecraft.entity.Entity.func_70091_d(Entity.java:952)
	at net.minecraft.entity.EntityLivingBase.func_191986_a(EntityLivingBase.java:1949)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2384)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
	at net.minecraft.entity.monster.EntityMob.func_70636_d(EntityMob.java:45)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2170)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(EntityMob.java:50)
	at net.minecraft.entity.monster.EntityCreeper.func_70071_h_(EntityCreeper.java:172)
	at net.daveyx0.primitivemobs.entity.monster.EntityPrimitiveCreeper.func_70071_h_(EntityPrimitiveCreeper.java:66)
	at net.daveyx0.primitivemobs.entity.monster.EntityRocketCreeper.func_70071_h_(EntityRocketCreeper.java:128)
	at net.minecraft.world.World.func_72866_a(World.java:1990)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1952)

-- Entity being ticked --
Details:
	Entity Type: primitivemobs:rocket_creeper (net.daveyx0.primitivemobs.entity.monster.EntityRocketCreeper)
	Entity ID: 1132239
	Entity Name: Rocket Creeper
	Entity's Exact location: -352.44, 61.00, 279.37
	Entity's Block location: World: (-353,61,279), Chunk: (at 15,3,7 in -23,17; contains blocks -368,0,272 to -353,255,287), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Entity's Momentum: -0.01, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1756)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)

-- Affected level --
Details:
	Level name: world
	All players: 4 total; [EntityPlayerMP['xbony2'/921781, l='world', x=-148.46, y=77.00, z=281.82], EntityPlayerMP['Lykrast'/898609, l='world', x=-286.70, y=16.80, z=257.53], EntityPlayerMP['Hubry'/997122, l='world', x=-79.67, y=19.00, z=65.37], EntityPlayerMP['DSquirrelGM'/904455, l='world', x=-354.70, y=60.80, z=279.58]]
	Chunk stats: ServerChunkCache: 1040 Drop: 0
	Level seed: -1196918858
	Level generator: ID 06 - BIOMESOP, ver 0. Features enabled: true
	Level generator options: 
	Level spawn location: World: (-156,64,256), Chunk: (at 4,4,0 in -10,16; contains blocks -160,0,256 to -145,255,271), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 1313000 game time, 1292885 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 94033 (now: false), thunder time: 114136 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Linux (amd64) version 2.6.32-504.23.4.el6.x86_64
	Java Version: 1.8.0_60, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2763357416 bytes (2635 MB) / 6336020480 bytes (6042 MB) up to 14989918208 bytes (14295 MB)
	JVM Flags: 0 total; 
	IntCache: cache: 0, tcache: 0, allocated: 3, tallocated: 104
	FML: MCP 9.42 Powered by Forge 14.23.4.2739 104 mods loaded, 104 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State     | ID                                | Version            | Source                                          | Signature                                |
	|:--------- |:--------------------------------- |:------------------ |:----------------------------------------------- |:---------------------------------------- |
	| UCHIJAAAA | minecraft                         | 1.12.2             | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | mcp                               | 9.42               | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | FML                               | 8.0.99.99          | forge-1.12.2-14.23.4.2739-universal.jar         | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | forge                             | 14.23.4.2739       | forge-1.12.2-14.23.4.2739-universal.jar         | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | incorporeal_core                  | 1.0.0              | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | OpenEye                           | 0.8                | OpenEye-1.12.2-0.8.jar                          | None                                     |
	| UCHIJAAAA | openmodscore                      | 0.11.5             | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | foamfixcore                       | 7.7.4              | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | opencomputers|core                | 1.7.2.67           | minecraft.jar                                   | None                                     |
	| UCHIJAAAA | ic2                               | 2.8.96-ex112       | industrialcraft-2-2.8.96-ex112.jar              | de041f9f6187debbc77034a344134053277aa3b0 |
	| UCHIJAAAA | advanced_solar_panels             | 4.2.0              | Advanced Solar Panels-4.2.0.jar                 | None                                     |
	| UCHIJAAAA | akashictome                       | 1.2-10             | AkashicTome-1.2-10.jar                          | None                                     |
	| UCHIJAAAA | biomesoplenty                     | 7.0.1.2399         | BiomesOPlenty-1.12.2-7.0.1.2399-universal.jar   | None                                     |
	| UCHIJAAAA | baubles                           | 1.5.2              | Baubles-1.12-1.5.2.jar                          | None                                     |
	| UCHIJAAAA | thaumcraft                        | 6.1.BETA22         | Thaumcraft-1.12.2-6.1.BETA22.jar                | None                                     |
	| UCHIJAAAA | botania                           | r1.10-354          | Botania r1.10-354.jar                           | None                                     |
	| UCHIJAAAA | craftstudioapi                    | 1.0.0              | CraftStudio-1.0.0.93-mc1.12-alpha.jar           | None                                     |
	| UCHIJAAAA | mantle                            | 1.12-1.3.2.24      | Mantle-1.12-1.3.2.24.jar                        | None                                     |
	| UCHIJAAAA | natura                            | 1.12.2-4.3.2.62    | natura-1.12.2-4.3.2.62.jar                      | None                                     |
	| UCHIJAAAA | codechickenlib                    | 3.2.1.351          | CodeChickenLib-1.12.2-3.2.1.351-universal.jar   | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UCHIJAAAA | redstoneflux                      | 2.0.2              | RedstoneFlux-1.12-2.0.2.3-universal.jar         | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | cofhcore                          | 4.5.3              | CoFHCore-1.12.2-4.5.3.20-universal.jar          | None                                     |
	| UCHIJAAAA | cofhworld                         | 1.2.0              | CoFHWorld-1.12.2-1.2.0.5-universal.jar          | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | thermalfoundation                 | 2.5.0              | ThermalFoundation-1.12.2-2.5.0.19-universal.jar | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | thermalexpansion                  | 5.5.0              | ThermalExpansion-1.12.2-5.5.0.29-universal.jar  | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | betterwithmods                    | ${version}         | BetterWithMods-1.12-2.3.8.jar                   | None                                     |
	| UCHIJAAAA | quark                             | r1.4-125           | Quark-r1.4-125.jar                              | None                                     |
	| UCHIJAAAA | animania                          | 1.5.1.1            | animania-1.12.2-1.5.1.1.jar                     | None                                     |
	| UCHIJAAAA | jei                               | 4.12.0.215         | jei_1.12.2-4.12.0.215.jar                       | None                                     |
	| UCHIJAAAA | appleskin                         | 1.0.9              | AppleSkin-mc1.12-1.0.9.jar                      | None                                     |
	| UCHIJAAAA | appliedenergistics2               | rv6-alpha-3        | appliedenergistics2-rv6-alpha-3.jar             | dfa4d3ac143316c6f32aa1a1beda1e34d42132e5 |
	| UCHIJAAAA | astralsorcery                     | 1.9.4              | astralsorcery-1.12.2-1.9.4.jar                  | a0f0b759d895c15ceb3e3bcb5f3c2db7c582edf0 |
	| UCHIJAAAA | attributefix                      | 1.0.1              | AttributeFix-1.12.2-1.0.1.jar                   | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCHIJAAAA | auginter                          | 1.1.0              | auginter-1.1.0-1.12.x.jar                       | None                                     |
	| UCHIJAAAA | psi                               | r1.1-59            | Psi-r1.1-59.jar                                 | None                                     |
	| UCHIJAAAA | autoreglib                        | 1.3-18             | AutoRegLib-1.3-18.jar                           | None                                     |
	| UCHIJAAAA | badwithernocookiereloaded         | 1.12.2-3.1.14      | badwithernocookiereloaded-1.12.2-3.1.14.jar     | None                                     |
	| UCHIJAAAA | bedpatch                          | 2.2                | bedpatch-2.2-1.12.2.jar                         | 6bf7527e690fb5e8719b9832bce5000a3e87dfe6 |
	| UCHIJAAAA | betterwithaddons                  | @VERSION@          | Better With Addons-0.45.jar                     | None                                     |
	| UCHIJAAAA | bibliocraft                       | 2.4.5              | BiblioCraft[v2.4.5][MC1.12.2].jar               | None                                     |
	| UCHIJAAAA | bonemealable_babies               | 1.2.0a             | bonemealablebabies-1.2.0a.jar                   | None                                     |
	| UCHIJAAAA | chameleon                         | 1.12-4.1.3         | Chameleon-1.12-4.1.3.jar                        | None                                     |
	| UCHIJAAAA | placebo                           | 1.4.1              | Placebo-1.12.2-1.4.1.jar                        | None                                     |
	| UCHIJAAAA | clickmachine                      | 1.1.3              | ClickMachine-1.12.2-1.1.3.jar                   | None                                     |
	| UCHIJAAAA | clumps                            | 3.1.1              | Clumps-3.1.1.jar                                | None                                     |
	| UCHIJAAAA | cyclicmagic                       | 1.16.9             | Cyclic-1.12.2-1.16.9.jar                        | 1bc8f8dbe770187a854cef35dad0ff40ba441bbe |
	| UCHIJAAAA | diethopper                        | 1.1                | diethopper-1.1.jar                              | None                                     |
	| UCHIJAAAA | discordintegration                | 3.0.4              | DiscordIntegration-mc1.12.2-3.0.5.jar           | None                                     |
	| UCHIJAAAA | ebwizardry                        | 4.1.2              | Electroblob's Wizardry - 4.1.2 - MC 1.12.2.jar  | None                                     |
	| UCHIJAAAA | endercore                         | 1.12.2-0.5.37      | EnderCore-1.12.2-0.5.37.jar                     | None                                     |
	| UCHIJAAAA | enderio                           | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiointegrationtic             | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiobase                       | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderioconduits                   | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderioconduitsappliedenergistics | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderioconduitsopencomputers      | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderioconduitsrefinedstorage     | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiointegrationforestry        | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiointegrationticlate         | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiomachines                   | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | enderiopowertools                 | 5.0.31             | EnderIO-1.12.2-5.0.31.jar                       | None                                     |
	| UCHIJAAAA | valkyrielib                       | 1.12.2-2.0.14a     | valkyrielib-1.12.2-2.0.14a.jar                  | None                                     |
	| UCHIJAAAA | environmentaltech                 | 1.12.2-2.0.14b     | environmentaltech-1.12.2-2.0.14b.jar            | None                                     |
	| UCHIJAAAA | sonarcore                         | 5.0.15             | sonarcore-1.12.2-5.0.15-13.jar                  | None                                     |
	| UCHIJAAAA | fluxnetworks                      | 3.0.16             | fluxnetworks-1.12.2-3.0.16-11.jar               | None                                     |
	| UCHIJAAAA | foamfix                           | 0.10.0-1.12.2      | foamfix-0.10.0-1.12.2.jar                       | None                                     |
	| UCHIJAAAA | forgemultipartcbe                 | 2.5.0.71           | ForgeMultipart-1.12.2-2.5.0.71-universal.jar    | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UCHIJAAAA | microblockcbe                     | 2.5.0.71           | ForgeMultipart-1.12.2-2.5.0.71-universal.jar    | None                                     |
	| UCHIJAAAA | minecraftmultipartcbe             | 2.5.0.71           | ForgeMultipart-1.12.2-2.5.0.71-universal.jar    | None                                     |
	| UCHIJAAAA | guideapi                          | 1.12-2.1.7-62      | Guide-API-1.12-2.1.7-62.jar                     | None                                     |
	| UCHIJAAAA | hardcoredarkness                  | 2.0                | HardcoreDarkness-MC1.12.2-2.0.jar               | d72e0dd57935b3e9476212aea0c0df352dd76291 |
	| UCHIJAAAA | ichunutil                         | 7.1.4              | iChunUtil-1.12.2-7.1.4.jar                      | None                                     |
	| UCHIJAAAA | incorporeal                       | 1.0.2              | incorporeal-1.0.2.jar                           | None                                     |
	| UCHIJAAAA | jaopca                            | 1.12.2-2.2.8.91    | JAOPCA-1.12.2-2.2.8.91.jar                      | None                                     |
	| UCHIJAAAA | oredictinit                       | 1.12.2-2.2.1.68    | JAOPCA-1.12.2-2.2.8.91.jar                      | None                                     |
	| UCHIJAAAA | lex                               | 1.0.6              | LibEx-1.12.2-1.0.6.jar                          | None                                     |
	| UCHIJAAAA | matteroverdrive                   | 0.6.0.176          | MatterOverdrive-1.12.2-0.6.0.176-universal.jar  | None                                     |
	| UCHIJAAAA | mcjtylib_ng                       | 3.0.3              | mcjtylib-1.12-3.0.3.jar                         | None                                     |
	| UCHIJAAAA | mobamputation                     | 7.0.0              | MobAmputation-1.12.2-7.0.0.jar                  | None                                     |
	| UCHIJAAAA | mysticalwildlife                  | 1.1.2              | mysticalwildlife-1.12.2-1.1.2.jar               | None                                     |
	| UCHIJAAAA | nex                               | 2.0.6              | NetherEx-1.12.2-2.0.6.jar                       | None                                     |
	| UCHIJAAAA | opencomputers                     | 1.7.2.67           | OpenComputers-MC1.12.2-1.7.2.67.jar             | None                                     |
	| UCHIJAAAA | xnet                              | 1.7.3              | xnet-1.12-1.7.3.jar                             | None                                     |
	| UCHIJAAAA | ocxnetdriver                      | 1.0.2              | ocxnetdriver-1.0.2-b11.jar                      | None                                     |
	| UCHIJAAAA | openmods                          | 0.11.5             | OpenModsLib-1.12.2-0.11.5.jar                   | d2a9a8e8440196e26a268d1f3ddc01b2e9c572a5 |
	| UCHIJAAAA | openblocks                        | 1.7.6              | OpenBlocks-1.12.2-1.7.6.jar                     | d2a9a8e8440196e26a268d1f3ddc01b2e9c572a5 |
	| UCHIJAAAA | primitivemobs                     | 1.1.6              | primitivemobs-1.1.6_beta.jar                    | None                                     |
	| UCHIJAAAA | prodigytech                       | 0.2.0              | prodigytech-1.12.2-0.2.0.jar                    | None                                     |
	| UCHIJAAAA | quickleafdecay                    | 1.2.4              | QuickLeafDecay-MC1.12.1-1.2.4.jar               | None                                     |
	| UCHIJAAAA | redstonearsenal                   | 2.5.1              | RedstoneArsenal-1.12.2-2.5.1.13-universal.jar   | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | stg                               | 1.12.2-1.2.3       | stg-1.12.2-1.2.3.jar                            | None                                     |
	| UCHIJAAAA | storagedrawers                    | 1.12-5.3.5         | StorageDrawers-1.12.2-5.3.7.jar                 | None                                     |
	| UCHIJAAAA | storagedrawersextra               | @VERSION@          | StorageDrawersExtras-1.12-3.1.0.jar             | None                                     |
	| UCHIJAAAA | thaumcomp                         | 0.3.0.16           | ThaumicComputers-MC1.12.2-0.3.0.16.jar          | None                                     |
	| UCHIJAAAA | thaumicterminal                   | 1.0.6              | ThaumicTerminal-1.0.6.jar                       | None                                     |
	| UCHIJAAAA | thaumictinkerer                   | 1.12.2-5.0-353c71c | thaumictinkerer-1.12.2-5.0-353c71c.jar          | None                                     |
	| UCHIJAAAA | theoneprobe                       | 1.4.23             | theoneprobe-1.12-1.4.23.jar                     | None                                     |
	| UCHIJAAAA | thermaldynamics                   | 2.5.1              | ThermalDynamics-1.12.2-2.5.1.14-universal.jar   | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | topaddons                         | 1.12.2-1.8.0       | topaddons-1.12.2-1.8.0.jar                      | None                                     |
	| UCHIJAAAA | translocators                     | 2.5.1.77           | Translocators-1.12.2-2.5.1.77-universal.jar     | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UCHIJAAAA | twilightforest                    | 3.7.424            | twilightforest-1.12.2-3.7.424-universal.jar     | None                                     |
	| UCHIJAAAA | universalmodifiers                | 1.12.2-1.0.9a      | valkyrielib-1.12.2-2.0.14a.jar                  | None                                     |
	| UCHIJAAAA | weirdinggadget                    | 1.0                | weirdinggadget-1.12.2-2.0.5-universal.jar       | None                                     |
	| UCHIJAAAA | wrapup                            | 1.12-1.1.3         | WrapUp-1.12-1.1.3.jar                           | None                                     |

	Loaded coremods (and transformers): 
Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.10.0-1.12.2.jar)
  pl.asie.foamfix.coremod.FoamFixTransformer
F5FixLoadingPlugin (f5fix-1.1.jar)
  de.morrien.f5fix.F5FixTransformer
EnderCorePlugin (EnderCore-1.12.2-0.5.37.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
  com.enderio.core.common.transform.SimpleMixinPatcher
OpenEyePlugin (OpenEye-1.12.2-0.8.jar)
  openeye.asm.MultiTransformer
BedPatch (bedpatch-2.2-1.12.2.jar)
  com.mordenkainen.bedpatch.BedPatchASM
AstralCore (astralsorcery-1.12.2-1.9.4.jar)
  
OpenModsCorePlugin (OpenModsLib-1.12.2-0.11.5.jar)
  openmods.core.OpenModsClassTransformer
CTMCorePlugin (CTM-MC1.12.2-0.3.2.18.jar)
  team.chisel.ctm.client.asm.CTMTransformer
LoadingPlugin (Quark-r1.4-125.jar)
  vazkii.quark.base.asm.ClassTransformer
Incorporeal Tweaker (incorporeal-1.0.2.jar)
  quaternary.incorporeal.spookyasm.IncorporealTransformer
LoadingPlugin (HardcoreDarkness-MC1.12.2-2.0.jar)
  lumien.hardcoredarkness.asm.ClassTransformer
TransformerLoader (OpenComputers-MC1.12.2-1.7.2.67.jar)
  li.cil.oc.common.asm.ClassTransformer
SmoothWaterCore (SmoothWater-0.1.4.jar)
  pl.asie.smoothwater.SmoothWaterTransformer
BetterFoliageLoader (BetterFoliage-MC1.12-2.1.10.jar)
  mods.betterfoliage.loader.BetterFoliageTransformer
	OpenModsLib class transformers: [llama_null_fix:FINISHED],[horse_base_null_fix:FINISHED],[pre_world_render_hook:ENABLED],[player_render_hook:ENABLED],[horse_null_fix:FINISHED]
	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaDecorative (Enabled/Not Forced)
		- NaturaTools (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)

	AE2 Version: alpha rv6-alpha-3 for Forge 14.23.3.2655
	Ender IO: No known problems detected.

	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
	!!!You are looking at the diagnostics information, not at the crash.       !!!
	!!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

	AE2 Integration: IC2:ON, RC:OFF, MFR:OFF, Waila:OFF, Mekanism:OFF, OpenComputers:ON, THE_ONE_PROBE:ON, TESLA:OFF, CRAFTTWEAKER:OFF
	Profiler Position: N/A (disabled)
	Player Count: 4 / 20; [EntityPlayerMP['xbony2'/921781, l='world', x=-148.46, y=77.00, z=281.82], EntityPlayerMP['Lykrast'/898609, l='world', x=-286.70, y=16.80, z=257.53], EntityPlayerMP['Hubry'/997122, l='world', x=-79.67, y=19.00, z=65.37], EntityPlayerMP['DSquirrelGM'/904455, l='world', x=-354.70, y=60.80, z=279.58]]
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)
```

**Comment by xbony2** [comment]
According to the PM author, this bug is now fixed.
